Simulator/player.py

```diff
@@ -1630,6 +1644,18 @@ def start_round(self, t_round):
         self.round = t_round
         self.reward += self.num_units_in_play * self.minion_count_reward
         self.gold_income(self.round)
+        for augment in self.augments:
+            handle_start_round_augment(self, augment)
+
+        # TODO: remove when augment is correctly implemented
+        if Augment.is_augment_round(self.round):
+            self.add_augment(Augment.AFK)
+        self.is_afk = False
```

Comment:
We already initiate the value to be false. Turning it to true is the odd case and probably should be done inside the Augment class instead of here.